File: app/src/main/java/com/my/televip/ui/ThemeColors.java
package com.my.televip.ui;

import static com.my.televip.virtuals.ActionBar.Theme.isLight;

import android.graphics.Color;

public class ThemeColors {


    public static int getBackgroundGrayColor(){
        return isLight() ? Color.rgb(241, 241, 243) : Color.rgb(21, 30, 39);
    }

    public static int getBackgroundWhiteOrBlueColor(){
        return isLight() ? Color.WHITE : Color.rgb(29, 39, 51);
    }

    public static int getToolBarColor(){
        return isLight() ? Color.WHITE : Color.rgb(36, 45, 57);
    }

    public static int getToolBarRippleColor(){
        return isLight() ? 0x20000000 : 0x20FFFFFF;
    }

    public static int getTextToolBarColor(){
        return isLight() ? Color.BLACK : Color.WHITE;
    }
    public static int getTextColor(){
        return isLight() ? Color.BLACK : Color.WHITE;
    }

    public static int getTextBlueColor(){
        return isLight() ? Color.rgb(100, 164, 221) : Color.rgb(112, 184, 221);
    }
    public static int getTextGrayColor(){
        return isLight() ? Color.rgb(128, 128, 128) : Color.rgb(103,115,128);
    }

    public static int getArrowDrawableColor(){
        return isLight() ? Color.BLACK : Color.WHITE;
    }

}

File: app/src/main/java/com/my/televip/ui/ThemeColors.kt
package com.my.televip.ui

import android.graphics.Color
import com.my.televip.virtuals.ActionBar.Theme

object ThemeColors {

    @JvmStatic
    fun getBackgroundGrayColor(): Int {
        return if (Theme.isLight()) Color.rgb(241, 241, 243) else Color.rgb(21, 30, 39)
    }

    @JvmStatic
    fun getBackgroundWhiteOrBlueColor(): Int {
        return if (Theme.isLight()) Color.WHITE else Color.rgb(29, 39, 51)
    }

    @JvmStatic
    fun getToolBarColor(): Int {
        return if (Theme.isLight()) Color.WHITE else Color.rgb(36, 45, 57)
    }

    @JvmStatic
    fun getToolBarRippleColor(): Int {
        return if (Theme.isLight()) 0x20000000 else 0x20FFFFFF.toInt()
    }

    @JvmStatic
    fun getTextToolBarColor(): Int {
        return if (Theme.isLight()) Color.BLACK else Color.WHITE
    }

    @JvmStatic
    fun getTextColor(): Int {
        return if (Theme.isLight()) Color.BLACK else Color.WHITE
    }

    @JvmStatic
    fun getTextBlueColor(): Int {
        return if (Theme.isLight()) Color.rgb(100, 164, 221) else Color.rgb(112, 184, 221)
    }

    @JvmStatic
    fun getTextGrayColor(): Int {
        return if (Theme.isLight()) Color.rgb(128, 128, 128) else Color.rgb(103, 115, 128)
    }

    @JvmStatic
    fun getArrowDrawableColor(): Int {
        return if (Theme.isLight()) Color.BLACK else Color.WHITE
    }
}
